build-tools/packages/build-cli/docs/generate.md

```diff
@@ -142,6 +143,29 @@ EXAMPLES
     $ flub generate changeset --all
 ```
 
+## `flub generate manifest`
+
+As all builds are published to build feed, this command picks the latest successful build against the provided branch name and generates dev manifest file.
+
+```
+USAGE
+  $ flub generate manifest -o <value> -p <value> -t <value> -s <value> [-v | --quiet]
```

Comment:
We call these "reports" elsewhere. This should be something like `flub release report --update` or `flub release report-update` or `flub release update-report`.

Reply:
Maybe `flub release report-latest`?

Reply:
or `flub release report-unreleased`?